wordsense/settings.py

```diff
@@ -156,3 +158,12 @@
 STATICFILES_DIRS = [os.path.join(BASE_DIR, 'build/static')]
 
 django_heroku.settings(locals())
+
+sentry_sdk.init(
+    dsn=os.environ['WS_SENTRY_DSN'],
```

Comment:
this would have a key error if it doesn't contain the key. suggesting using
`os.get('WS_SENTRY_DSN', <a default value>)`

Reply:
Sentry needs its project specific url, so I don't think that there is a point. I will add the value to Heroku env vars